test/integration_testproj.go

```diff
@@ -23,41 +29,53 @@ type IntegrationTestProject struct {
 	t          *testing.T
 	h          *Helper
 	preImports []string
+	tempdir    string
+	env        []string
+	origWd     string
+	stdout     bytes.Buffer
+	stderr     bytes.Buffer
 }
 
-func NewTestProject(t *testing.T, initPath string) *IntegrationTestProject {
+func NewTestProject(t *testing.T, initPath, wd string) *IntegrationTestProject {
 	new := &IntegrationTestProject{
-		t: t,
-		h: NewHelper(t),
+		t:      t,
+		origWd: wd,
+		env:    os.Environ(),
 	}
-	new.TempDir(ProjectRoot)
+	new.makeRootTempDir()
 	new.TempDir(ProjectRoot, "vendor")
 	new.CopyTree(initPath)
-	new.h.Setenv("GOPATH", new.h.Path("."))
-	new.h.Cd(new.Path(ProjectRoot))
+	if runtime.GOOS == "darwin" {
+		new.Setenv("GOPATH", filepath.Join("/private", new.tempdir))
```

Comment:
Yes, it's a thing - idk why, but in Apple's (bsd's?) infinite wisdom, they decided it's a good idea for `/var`, `/etc`, and `/tmp` to be symlinks to that respective name under a `/private` dir. Maybe it's a permissioning thing. 

So, right - I meant, only do that extra check if we're on `darwin`.